test/lib/mocked-dockerode.ts

```diff
@@ -62,18 +80,28 @@ export interface TestData {
 function createMockedDockerode(data: TestData) {
 	const mockedDockerode = dockerode.prototype;
 	mockedDockerode.getNetwork = (id: string) => {
+		addAction('getNetwork', { id });
 		return {
 			inspect: async () => {
+				addAction('inspect', {});
 				return data.networks[id];
 			},
 		} as dockerode.Network;
 	};
 
 	mockedDockerode.getImage = (name: string) => {
+		addAction('getImage', { name });
```

Comment:
I couldn't think of a good way to add a Spy on this so had to create this...if anyone knows a better way let me know.